Vagrantfile

```diff
@@ -1,6 +1,6 @@
 Vagrant.configure("2") do |config|
   config.vm.box = "fredrikhgrelland/hashistack"
-  config.vm.box_version = "~> 0.1"
+  config.vm.box_version = "0.1.0"
```

Comment:
but this 0.1.0 is still giving error.
 Bringing machine 'default' up with 'virtualbox' provider...
==> default: Box 'fredrikhgrelland/hashistack' could not be found. Attempting to find and install...
    default: Box Provider: virtualbox
    default: Box Version: 0.1.0
The box 'fredrikhgrelland/hashistack' could not be found or
could not be accessed in the remote catalog. If this is a private
box on HashiCorp's Vagrant Cloud, please verify you're logged in via
`vagrant login`. Also, please double-check the name. The expanded
URL and error message are shown below:


This is working with config.vm.box_version = "0.1.1"

Reply:
That is weird.
1. Tests are ok log is here https://github.com/fredrikhgrelland/data-mesh/runs/825477778#step:3:15 
2. Also I run it locally, works fine

Reply:
![88](https://user-images.githubusercontent.com/15572799/86355548-21a1dd00-bc6b-11ea-8097-e5052519a7b1.PNG)